### ISSUE TITLE
`$pagination` variable only available when defined in "top level" `.blade.php` files

### ISSUE THREAD

**Issue by amadeann** [tightenco/jigsaw#474]
I structure my Jigsaw sites as this:

```
source
    ...
    _templates
        ...
        about.blade.php
        blog.blade.php
        simple.blade.php
        ...
    ...
    about.blade.md
    blog.blade.md
    index.blade.md
    simple-landing-page.blade.php
    ...
```
`/about` extends `_templates.about`, `/blog` extends `_templates.blog` and `/` and `/simple-landing-page` extends both extend `_templates.simple`. All templates extend `_layouts.master`.

Some reasons why I do it this way:
1. Reuse common templates
2. Localization (as I described in the comment to #25 )
3. Separate the page layout from the content/metadata

With that setup, `$pagination` variable is not available when I include it in `blog.blade.md` (getting `Notice: Undefined variable: pagination in`). I suspect the reason behind this is that
`PaginatedPageHandler@shouldHandle` method checks whether the file name ends with `blade.php`, but simply commenting that out didn't help.

**Comment by damiani** [comment]
`PaginatedPageHandler` was indeed the culprit, but needed an adjustment in two places to accommodate `.blade.md` files. I don't thing the "level" of the template was the issue, but the fact that it wasn't a pure Blade file. This should be fixed now in the latest version, v1.3.28.

**Comment by amadeann** [comment]
> `PaginatedPageHandler` was indeed the culprit, but needed an adjustment in two places to accommodate `.blade.md` files. I don't thing the "level" of the template was the issue, but the fact that it wasn't a pure Blade file. This should be fixed now in the latest version, v1.3.28.

Works for me now :) Thank you for the fix!
